res/controllers/Ion-Discover-DJ-Pro.midi.xml

```diff
@@ -0,0 +1,797 @@
+<?xml version='1.0' encoding='utf-8'?>
+<MixxxControllerPreset mixxxVersion="2.1.0+" schemaVersion="1">
+    <info>
+        <name>Ion Discover DJ Pro</name>
+        <author>joachim,meltedpianoman</author>
+        <description>This preset provides functionality for the Ion Discover DJ Ppro controller for Mixxx</description>
+        <forums>https://mixxx.org/forums/viewtopic.php?f=7&amp;t=3343</forums>
+    </info>

```

Comment:
Please add a wiki page and put the link here.

Reply:
https://mixxx.org/wiki/doku.php/ion_discover_dj_pro